2021-08-18 15:50:07,984 INFO    MainThread:8018 [wandb_setup.py:_flush():69] setting env: {}
2021-08-18 15:50:07,984 INFO    MainThread:8018 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-08-18 15:50:07,985 INFO    MainThread:8018 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-08-18 15:50:07,985 INFO    MainThread:8018 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-18 15:50:07,985 INFO    MainThread:8018 [wandb_init.py:_log_setup():342] Logging user logs to /root/konst/ml-utils/training/edm_genre_classify/wandb/run-20210818_155007-45hr79jz/logs/debug.log
2021-08-18 15:50:07,985 INFO    MainThread:8018 [wandb_init.py:_log_setup():343] Logging internal logs to /root/konst/ml-utils/training/edm_genre_classify/wandb/run-20210818_155007-45hr79jz/logs/debug-internal.log
2021-08-18 15:50:07,985 INFO    MainThread:8018 [wandb_init.py:init():375] calling init triggers
2021-08-18 15:50:07,985 INFO    MainThread:8018 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/melspects_general.npz', 'checkpoint_path': 'model_checkpoints/', 'model_path': '/root/konst/ml-utils/training/edm_genre_classify/model_checkpoints/edm_genre_epoch_16_acc_0.9111_vs_0.6955_loss_6.3745_vs_6.8012.pth', 'batch_size': 20, 'epochs': 100, 'start_epoch': 17, 'lam': 0.04, 'patience': 2, 'factor': 0.8, 'use_wandb': True, 'lr': 0.001, 'save_at': 1, 'adjust_lr_every': 1, 'resize': 180, 'num_workers': 1}
2021-08-18 15:50:07,985 INFO    MainThread:8018 [wandb_init.py:init():424] starting backend
2021-08-18 15:50:07,985 INFO    MainThread:8018 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-18 15:50:08,001 INFO    MainThread:8018 [backend.py:ensure_launched():135] starting backend process...
2021-08-18 15:50:08,008 INFO    MainThread:8018 [backend.py:ensure_launched():140] started backend process with pid: 8036
2021-08-18 15:50:08,009 INFO    MainThread:8018 [wandb_init.py:init():429] backend started and connected
2021-08-18 15:50:08,011 INFO    MainThread:8018 [wandb_init.py:init():477] updated telemetry
2021-08-18 15:50:08,011 INFO    MainThread:8018 [wandb_init.py:init():500] communicating current version
2021-08-18 15:50:08,452 INFO    MainThread:8018 [wandb_init.py:init():505] got version response 
2021-08-18 15:50:08,452 INFO    MainThread:8018 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-18 15:50:08,644 INFO    MainThread:8018 [wandb_init.py:init():540] starting run threads in backend
2021-08-18 15:50:09,513 INFO    MainThread:8018 [wandb_run.py:_console_start():1584] atexit reg
2021-08-18 15:50:09,513 INFO    MainThread:8018 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-08-18 15:50:09,514 INFO    MainThread:8018 [wandb_run.py:_redirect():1463] Redirecting console.
2021-08-18 15:50:09,515 INFO    MainThread:8018 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-18 15:50:09,515 INFO    MainThread:8018 [wandb_init.py:init():565] run started, returning control to user process
2021-08-19 00:48:46,825 INFO    MainThread:8018 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 255
2021-08-19 00:48:46,827 INFO    MainThread:8018 [wandb_run.py:_restore():1526] restore
